CONTRIBUTING.md

```diff
@@ -114,6 +114,7 @@ Once you have babel running in watch mode, you can start making changes to parse
 * Take testing seriously! Aim to increase the test coverage with every pull request. To obtain the test coverage of the project, run: `npm run coverage`
 * Run the tests for the file you are working on with the following command: `npm test spec/MyFile.spec.js`
 * Run the tests for the whole project to make sure the code passes all tests. This can be done by running the test command for a single file but removing the test file argument. The results can be seen at *<PROJECT_ROOT>/coverage/lcov-report/index.html*.
+* Format your code by running `npm run prettier` followed by `npm run lint-fix`.
```

Comment:
We could update the prettier command in `package.json` and add lint-fix to it so that this is one step instead of two.

Reply:
To chime in, we have had this discussion some time ago and I suggested to keep those two apart, to be able to run either one. Maybe a `clean` script could execute both?